content/code-style/css.md

```diff
@@ -1,6 +1,6 @@
 # CSS Style Guide
 
-- [Preprocessing](#preprosessing)
+- [Preprocessing](#preprocessing)
```

Comment:
Nice catches!

If you spot some more, open up a separate PR for typos, so I can merge that it independently of the csp changes.

Leave these as they are for now.

Reply:
I will definitely think about it next time.